solana/program_v2/packages/client/core/src/GatekeeperService.ts

```diff
@@ -149,12 +149,12 @@ export class GatekeeperService extends AbstractService {
   issue(
     passAccount: PublicKey,
     subject: PublicKey,
-    splToken?: PublicKey,
```

Comment:
Nothing we should change now, but do you feel we need to pass this through?... it's always going to be the same SPL token program right?

Reply:
I think it will change if we use token 2022 so customers can specify? But agree it could be a flag or assumed by the ATAs?

Reply:
Yeah I had a similar thought about being able to specify any... this would have an implication on the program side though right? Or would the spl token library support both? (just curious)

Reply:
anchor-spl should report both need to take a look into that.